### ISSUE TITLE
GetNeighbourObjArr missing, breadcrumbs tree empty

### ISSUE THREAD

**Issue by krzemian** [SkepticMystic/breadcrumbs#290]
Hey

I never got Breadcrumbs to work, now troubleshooting it.

A sample note with
> Parent:: [[Cooking MOC]]

As the first line

DataView installed and enabled
Breadcrumbs plugin installed and enabled

Tried debugging it in the DEBUG mode, but console shows no errors. Interestingly, there's also no GetNeighbourObjArr when I refresh the index.

What's the next step?

**Comment by SkepticMystic** [comment]
Hey, thanks for checking the debug logs :)

It may be that you need to add the `parent` field to your Hierarchies. It isn't recognized by Breadcrumbs by default

**Comment by krzemian** [comment]
True, that was it, thanks. It didn't strike me it would not be set up by
default especially that I believe some of the examples used "parent".
Anyway, thanks a lot for a quick response.

śr., 26 sty 2022 o 06:24 SkepticMystic ***@***.***>
napisał(a):
